### PR TITLE
update to fody 2

### LINKED ISSUE #13: Update to Fody Version 2
Fody Version 2 is out and you will need to do an update for your addin to be usable with it

Specifically:

 * Update to the newest Fody.Cecil nuget
 * Change the min dependency ranged for Fody to be at least 2.0

 Please ping me if u need help

### PR DESCRIPTION
**Pull request by SimonCropp**
and updated other refs

fixes #13

### CODE REVIEW

**Comment by SimonCropp** [comment]
@distantcam this works locally for me. any ideas on the appveyor failure

**Comment by distantcam** [comment]
@SimonCropp You updated nunit. You'll have to update the appveyor file to run nunit 3.

See https://github.com/Fody/Costura/blob/master/appveyor.yml#L18

**Comment by cremor** [comment]
@SimonCropp Thanks for your work on this! I also started to work on a pull request for this today and only now saw that you already submitted one  :neutral_face:

One thing I noticed in your pull request: You added 3 \*.received.txt ApprovalTests files. Those shouldn't be in the repo.

**Comment by SimonCropp** [comment]
:shipit:

File: .gitignore [hunk 1]
```diff
@@ -66,6 +66,8 @@ dlldata.c
 *.svclog
 *.scc
 
+*.received.txt
```
Comment (distantcam) [inline .gitignore]: Why? I like having received files turn up in the git status.

**Comment by SimonCropp** [comment]
@distantcam fixed

**Pull request merged**